src/status_update.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+export default function statusUpdate(tasks) {
+  const checkbox = document.querySelectorAll('input[type=checkbox]');
+  const descriptionSpans = document.getElementsByClassName('text');
+  for (let i = 0; i < checkbox.length; i += 1) {
+    checkbox[i].addEventListener('click', () => {
+      if (checkbox[i].checked) {
+        tasks[i].completed = true;
+        descriptionSpans[i].classList.add('check');
+        localStorage.setItem('items', JSON.stringify(tasks));
+      } else {
+        tasks[i].completed = false;
+        descriptionSpans[i].classList.remove('check');
+        localStorage.setItem('items', JSON.stringify(tasks));
+      }
+    });
+  }
```

Comment:
* Kindly make use of this method to handle the changing status functionality, as currently this function is not been used in the `index.js` file. 
* This is important as this project is meant to give more exposure to the use of ES6 modules to write modular JavaScript [Check learning objectives](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Use%20ES6%20modules%20to%20write%20modular%20JavaScript)